### DISCUSSION TITLE
Question about SSH connection

### DISCUSSION THREAD

**Discussion by lucasmlrd** [gravitational/teleport#37272] [Q&A] [answered]
Hello,

I've just installed an open source teleport to test this solution. I'd like it to connect via ssh to some equipment (notably Fortigate). Is it possible to add a resource that would ask us for the password to access the cli via teleport?

Thanks in advance.

**Comment by stevenGravy** [comment, marked as answer]
Teleport does not support connecting with via password authentication for its SSH protocol support. You could use the TCP app access to provide network connectivity

**Comment by lucasmlrd** [comment]
Thank you for your reply. Is there a way to connect ssh using an rsa key via teleport?

Thank you

**Reply by stevenGravy** [reply]
If you have a rsa key and connect with Teleport through a machine you should be able to use that rsa key via agent forwarding. Direct thru the proxy service no.

**Reply by lucasmlrd** [reply]
Ok thanks for ur answer ! Have a good day !
